go.mod

```diff
@@ -1,28 +1,31 @@
 module github.com/evmos/evmos/v15
 
-go 1.20
+go 1.21
 
 require (
-	cosmossdk.io/api v0.3.1
+	cosmossdk.io/api v0.7.3-0.20231029200940-6af7f30bfd54
 	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
 	cosmossdk.io/math v1.2.0
-	cosmossdk.io/simapp v0.0.0-20230608160436-666c345ad23d
+	cosmossdk.io/simapp v0.0.0-20231107171416-e36a25c65b33
```

Comment:
most items that were within simapp have been moved to testutil. Using simapp, which is untagged may cause issues for you?

Reply:
removed this dependency, thanks!